web3/src/main/java/org/hiero/mirror/web3/evm/properties/MirrorNodeEvmProperties.java

```diff
@@ -199,6 +199,9 @@ public class MirrorNodeEvmProperties implements EvmProperties {
     @DecimalMax("1.0")
     private double modularizedTrafficPercent = 0.0;
 
+    @Getter
+    private long entityIdReservationHeadroom = 10L;
```

Comment:
Maybe a simpler name like
```suggestion
    private long entityNumBuffer = 1000L;
```

Reply:
Addressed